lib/graphql/argument.rb

```diff
@@ -94,6 +94,16 @@ def prepare=(prepare_proc)
       @prepare_proc = BackwardsCompatibility.wrap_arity(prepare_proc, from: 1, to: 2, name: "Argument#prepare(value, ctx)")
     end
 
+    def to_ast_node
```

Comment:
Are these dangerous to memoize? I wouldn't think so with `redefine` but maybe 🤔 ?

Reply:
The issue is that the _ast nodes_ are mutable. There's really no way to work with the AST right now except to modify nodes, so the cache's contents could become invalid. Can this be a "cross that bridge when we come to it" feature? Or, is there a need to cache the output now?